drivers/scheduler/k8s/k8s.go

```diff
@@ -5177,6 +5276,62 @@ func (k *K8s) createSnapshotAndVerify(data, snapName, namespace, snapClass, rest
 	return nil
 }
 
+func (k *K8s) cloneAndVerify(data, namespace, storageClass, clonedPVCName, originalPod, originalPVC string) error {
+	clientset, err := getKubeClient("")
+	if err != nil {
+		logrus.Errorf("Failed to get kube client for clone test: %s", err)
+		return err
+	}
+	cmdArgs2 := []string{"exec", "-it", originalPod, "-n", namespace, "--", "/bin/sh", "-c", fmt.Sprintf("echo -n %s >>  /mnt/volume1/aaaa.txt", data)}
```

Comment:
Can we move the original data write out into a different method and call it before this? That way this method only does the "cloning and verifying", and the setup is done beforehand.
(This also opens us up for running multiple clones in parallel in this test)

Reply:
Agreed.